src/generators/modelsGenerator.ts

```diff
@@ -585,9 +585,11 @@ function getProperties(
       )}"`;
     }
 
-    return property.name === "siblings"
-      ? `${(objectDetails as PolymorphicObjectDetails).unionName}[]`
-      : property.type;
+    const typeName =
+      property.name === "siblings"
+        ? `${(objectDetails as PolymorphicObjectDetails).unionName}[]`
+        : property.type;
+    return property.nullable ? typeName + " | null" : typeName;

```

Comment:
nit: you can also use string interpolation, I personally like it better for readability. However it is just personal taste

```typescript
return property.nullable ? `${typeName} | null` : typeName;
```